NEWS.md

```diff
@@ -27,10 +27,14 @@
 
 ## New and improved features
 
+* Quarto files (`.qmd`) were supported out of the box. The documentation and the 
```

Comment:
Since it's not really a new feature, I would put this under a `## Notes` section.

Also `s/ascertain/reflect/`

Reply:
I put this under `## Other Changes`. Is that okay?